test/unit/mocks.js

```diff
@@ -114,6 +146,44 @@ const tagMock = {
   name: 'mock tag'
 }
 
+const entryWithResourceLinksMock = {
+  sys: Object.assign(copy(sysMock), {
+    type: 'Entry',
+    contentType: Object.assign(copy(linkMock), { linkType: 'ContentType' }),
+    locale: 'locale'
+  }),
+  fields: {
+    xspace: [
```

Comment:
Can we have a field in the non-array form? Just to cover more
Maybe also an entry with multiple locales
```
  "fields": {
    "oneresource": {
      "en-US": {
        "sys": {
          "type": "ResourceLink",
          "linkType": "Contentful:Entry",
          "urn": "..."
        }
      }
    },
...
```